fs-one-concepts.md

```diff
@@ -4,18 +4,62 @@ Try your best to answer each question on your own before looking up the answer o
 
 #### 1. What is Enzyme and what are some of the methods that it provides?
 
+Enzyme allows for you to do testimg while using react by making it easier to test your react components output data. It allows for you to manipulate and simulate runtime given the output.
```

Comment:
I think you're getting the idea here. What you want to remember is that Enzyme makes it possible to find and interact with elements on your page.